.gitignore

```diff
@@ -101,3 +101,4 @@ gmailLogin.properties
 webLogin.properties
 mailHandlerTestProps.properties
 config.yaml
+mturk.properties
```

Comment:
Gmail, Web, Mturk, … Sums up. Just add them to a `test` section in the main file? Or have one `test.yml` that contains all tests configs.